### PR TITLE
Step 2 and 3: To handle a recipient for a user message

### PR DESCRIPTION
**Pull request by PrinceUmbrella**
Just made minor changes to send a message to the recipient step 2 and 3

### CODE REVIEW

**Comment by PrinceUmbrella** [comment]
If you can you remove the first four commits

**Comment by PrinceUmbrella** [comment]
Hi Everyone, I had some trouble committing that why you see 4 commits here. But the last one has the correct changes that were supposed to be made. I made changes so that it can handle the recipient. I deleted the `showMessageIfLoggedIn() ` so that @crepricg can add it for her feature.

**Review by pragyeenepal — APPROVED** [review]

**Review by pragyeenepal — APPROVED** [review]

**Review by pragyeenepal — APPROVED** [review]

**Review by pragyeenepal — APPROVED** [review]

**Review by pragyeenepal — APPROVED** [review]

**Review by pragyeenepal — APPROVED** [review]

**Review by pragyeenepal — APPROVED** [review]

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 1]
```diff
@@ -52,23 +53,25 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
```
Comment (kmontgomery) [inline src/main/java/com/google/codeu/data/Datastore.java:55]: This is an example.

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -52,23 +53,25 @@ public void storeMessage(Message message) {
...
+            new Query("Message")
+                    .setFilter(new Query.FilterPredicate("recipient", FilterOperator.EQUAL, recipient))
+                    .addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
 
     for (Entity entity : results.asIterable()) {
       try {
         String idString = entity.getKey().getName();
         UUID id = UUID.fromString(idString);
+        String user = (String) entity.getProperty("user");
```
Comment (kmontgomery) [inline src/main/java/com/google/codeu/data/Datastore.java:69]: nice work here!

**Pull request merged**